src/utils.ts

```diff
@@ -25,6 +45,49 @@ export const assert = (cond: any, message?: string | undefined): void => {
     }
 };
 
-export function noop(): void {
-    // noop
-}
+export const values = <T = any>(value: Dictionary<T>): T[] => {
+    if (isFunction(Object.values)) {
+        return Object.values(value);
+    }
+
+    return Object.keys(value).map(key => value[key]);
+};
+
+export const flat = <T = any>(value: T[][]): T[] => (
```

Comment:
Nit: The name `flatten` is more standard than `flat`

Reply:
[`flat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat) is the new array method, I decided that is acceptable to use the same name.